src/initialStore.js

```diff
@@ -1,7 +1,461 @@
 const initialStore = {
```

Comment:
Is currently not imported anywhere. Should probably build out a feature to store this information in a JSON file and parse it into the questionnaire.